docs/monitoring.md

```diff
@@ -0,0 +1,221 @@
+        {
+            "metrics":
+            {
+                "enable" : true,
+                "metrics_list": ["ovms_requests_success", "ovms_infer_req_queue_size"]
+            }
+        }
+    }
+    ```
+
+## Available metrics families
+
+Metrics from default list are enabled with the metrics_enabled flag or json configuration.
+
+However, you can enable also additional metrics by listing all the metrics you want to enable in the metric_list flag or json configuration.
+
+DEFAULT
+
+   ```bash
+        ovms_streams
+        ovms_current_requests
+        ovms_requests_success
+        ovms_requests_fail
+        ovms_request_time_us
+        ovms_inference_time_us
+        ovms_wait_for_infer_req_time_us
+   ```
+
+ADDITIONAL
+
+   ```bash
+        ovms_infer_req_queue_size
+        ovms_infer_req_active
+   ```
+
+## List of available metrics per interface
+
+Metrics from default list are enabled with the metrics_enabled flag or json configuration.
+
+However, you can enable also additional metrics by listing all the metrics you want to enable in the metric_list flag or json configuration.
+
+ovms_requests_success
+
+   ```bash
+        ovms_requests_success_grpc_predict
+        ovms_requests_success_grpc_getmodelmetadata
+        ovms_requests_success_grpc_getmodelstatus
+        ovms_requests_success_grpc_modelinfer
+        ovms_requests_success_grpc_modelmetadata
+        ovms_requests_success_grpc_modelready
+        ovms_requests_success_rest_modelinfer
+        ovms_requests_success_rest_predict
+        ovms_requests_success_rest_modelmetadata
+        ovms_requests_success_rest_modelstatus
+        ovms_requests_success_rest_modelmetadata
+        ovms_requests_success_rest_modelready
+   ```
+
+ovms_requests_fail
+
+   ```bash
+        ovms_requests_fail_grpc_predict
+        ovms_requests_fail_grpc_getmodelmetadata
+        ovms_requests_fail_grpc_getmodelstatus
+        ovms_requests_fail_grpc_modelinfer
+        ovms_requests_fail_grpc_modelmetadata
+        ovms_requests_fail_grpc_modelready
+        ovms_requests_fail_rest_modelinfer
+        ovms_requests_fail_rest_predict
+        ovms_requests_fail_rest_modelmetadata
+        ovms_requests_fail_rest_modelstatus
+        ovms_requests_fail_rest_modelmetadata
+        ovms_requests_fail_rest_modelready
+   ```
+
+ovms_request_time_us
+
+   ```bash
+        ovms_request_time_us_grpc
+        ovms_request_time_us_rest
+   ```
+
+## Changing metric endpoint port via CLI
+
+Currently the metrics endpoint is located on the '''rest_port/metrics''' endpoint.
+
+Changing the default setting is not part of the preview feature.
+
+## Example response from metrics endpoint
+
+TODO
+
+## Example graphics from Graphana
+
+TODO
+
+## Alternative for internal OVMS metrics: 
+
+Using 3rd party components acting as proxy between the clients and the server. 
+
+Examples are HAproxy, IstIo mesh, openshift mesh, Kubernetes ingress controller. 
+
+## Hardware specific metrics
```

Comment:
Let's drop this section. For now there are no HW exporters to offer

Reply:
Ok